### PR TITLE
fix stale fixture cache by invalidating on bulk delete

### PR DESCRIPTION
**Pull request by dannyroberts**
@benrudolph cc: @kaapstorm 

Fixes http://manage.dimagi.com/default.asp?220995 introduced by https://github.com/dimagi/commcare-hq/pull/10699

### CODE REVIEW

File: corehq/apps/cachehq/mixins.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import logging
 from django.conf import settings
-from couchdbkit import ResourceNotFound
+from couchdbkit import ResourceNotFound, Document
```
Comment (dimagimon) [inline corehq/apps/cachehq/mixins.py]: F401 'Document' imported but unused

File: corehq/apps/cachehq/mixins.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 from corehq.apps.cachehq.invalidate import invalidate_document
+from corehq.doctypemigrations.continuous_migrate import delete_docs
```
Comment (dimagimon) [inline corehq/apps/cachehq/mixins.py]: F401 'delete_docs' imported but unused

File: corehq/apps/cachehq/mixins.py [hunk 3]
```diff
@@ -34,6 +37,16 @@ def delete(self):
         self.clear_caches()
         invalidate_document(self, couch_db=self.get_db(), deleted=True)
 
+    @classmethod
```
Comment (dimagimon) [inline corehq/apps/cachehq/mixins.py]: F811 redefinition of unused 'delete_docs' from line 5

**Comment by dannyroberts** [comment]
:white_check_mark:

**Comment by czue** [comment]
nice, looks good. how's the test coverage for this class? seems like something that would be worth having good coverage for

**Comment by dannyroberts** [comment]
There's one test, but it doesn't really try to cover all the cases. Let me take a shot at it

**Comment by dannyroberts** [comment]
I went ahead and added a test that's mostly relevant to this PR: https://github.com/dimagi/commcare-hq/pull/10786

**Pull request merged**